<?xml version="1.0"?>
<launch>
    <node pkg="waypoint_updater" type="waypoint_updater.py" name="waypoint_updater">
      <param name="max_velocity" value="4.4"/>
      <param name="stop_distance" value="5.0"/>
      <param name="max_accel" value="1.0"/>
      <param name="loop" value="true" type="bool"/>
    </node>
</launch>
